server/views/pages/bookAVisit/checkYourBooking.njk

```diff
@@ -138,7 +138,7 @@
             text: "Main contact"
           },
           value: {
-            html: '<p><span class="test-main-contact-name">' + contactName | escape+ '</span><br><span class="test-main-contact-number">' +
+            html: '<p><span class="test-main-contact-name">' + contactName + '</span><br><span class="test-main-contact-number">' +
```

Comment:
No need to `escape` here because it is auto-escaped when the variable is defined on line 32 above.